### ISSUE TITLE
Please build a new PyPi release

### ISSUE THREAD

**Issue by athornton** [mogthesprog/jwtauthenticator#27]
...maybe after Yuvi's suggestion of putting the whole JWT into auth_state.

**Comment by groodt** [comment]
Yes, I'd also appreciate a new version. I've just realised the version on pypi as ancient and doesn't support configuring `header_is_authorization`.
